workbench/app/Console/StatCommand.php

```
@@ -4,115 +4,222 @@
-    protected $signature = 'nusa:stat
-                            {--d|diff : Generate diff from existing stats}
-                            {--w|write : Write latest stat to file}';
+    protected $signature = 'nusa:stat';
 
-    protected $description = 'Generate stats after test';
+    protected $description = 'Generate stats of imported database from upstream';
 
     /**
      * Execute the console command.
      */
     public function handle(Province $province): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$province'. (Unused Code Rules)

(UnusedFormalParameter)